### PR TITLE
feat(CX-3224): update settings header

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-3224]

### Description

| Web | mWeb |
| --- | --- |
| image | image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3224]: https://artsyproduct.atlassian.net/browse/CX-3224?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olerichter00 — APPROVED** [review]
Looks good 🌟

File: src/Apps/Settings/SettingsApp.tsx [hunk 1]
```diff
@@ -51,9 +53,28 @@ const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
...
+              displayBackArrow
+              useWithoutSeparator
+              href="/collector-profile" // TODO: use onClick and router or
+              // something else to make sure we're coming back to the correct
+              // tab of collector profile
+            >
+              Back to Collector Profile
+            </TopContextBar>
+          </Media>
+          <Text variant={["lg-display", "lg-display"]} mb={[4, 6]} mt={[2, 4]}>
```
Comment (MounirDhahri) [inline src/Apps/Settings/SettingsApp.tsx:69]:
```suggestion
          <Text variant="lg-display"} mb={[4, 6]} mt={[2, 4]}>
```

File: src/Apps/Settings/SettingsApp.tsx [hunk 2]
```diff
@@ -51,9 +53,28 @@ const SettingsApp: React.FC<SettingsAppProps> = ({ me, children }) => {
...
-      <Text variant={["lg-display", "xl"]} mt={4}>
-        Hi, {me.name}
-      </Text>
+      {isCollectorProfileEnabled ? (
+        <>
+          <Media greaterThan="xs">
+            <TopContextBar
+              displayBackArrow
+              useWithoutSeparator
+              href="/collector-profile" // TODO: use onClick and router or
```
Comment (MounirDhahri) [inline src/Apps/Settings/SettingsApp.tsx:62]: **thought:** do we have a ticket for this 👀

File: src/Components/TopContextBar.tsx [hunk 3]
```diff
@@ -85,9 +87,11 @@ export const TopContextBar: React.FC<TopContextBarProps> = ({
         {rightContent}
       </Flex>
 
-      <FullBleed>
-        <Separator as="hr" color="black15" />
-      </FullBleed>
+      {!useWithoutSeparator && (
```
Comment (MounirDhahri) [inline src/Components/TopContextBar.tsx:90]:
```suggestion
      {!!hideSeparator && (
```

File: src/Components/TopContextBar.tsx [hunk 4]
```diff
@@ -29,6 +30,7 @@ export const TopContextBar: React.FC<TopContextBarProps> = ({
   src,
   rightContent,
   onClick,
+  useWithoutSeparator,
```
Comment (MounirDhahri) [inline src/Components/TopContextBar.tsx:33]:
```suggestion
  hideSeparator = false,
```

File: src/Apps/Settings/SettingsApp.tsx (hunk 2 above)
Comment (dariakoko) [reply src/Apps/Settings/SettingsApp.tsx:62]: Nope, it's just a reminder, as 'nice to have' most probaly

**Comment by dariakoko** [comment]
@MounirDhahri I'll addredd your comments in another PR, was too fast to merge it today

**Pull request merged**